linkers/d-x86_64-openwrt-linux-musl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# This forces export of needed binding for Java
+LINKER=$(basename $0)
+LINKER=${LINKER#"d-"}
+ARGS=$@
+
+while (( "$#" )); do
+	if [[ $1 == *"-Wl,--version-script="* && ! $ARGS =~ -o.*librustls_platform_verifier.* ]]; then
+		VERSION=${1#"-Wl,--version-script="}
+		SED_ARGS='s/global:/global:\n    Java*;\n    JNI_OnLoad*;\n    _wrap*;\n/'
```

Comment:
OpenWRT probably will not ever need Java symbols :) 
```suggestion
		SED_ARGS='s/global:/global:\n    _wrap*;\n/'
```

Reply:
Just copied out from the others, but yeah, will adjust